skins/oasis/modules/OasisController.class.php

```diff
@@ -350,48 +267,28 @@ private function loadJs() {
 			$assetGroups[] = 'oasis_anon_js';
 		}
 
-
-		$jsLoader = '';
-
 		wfRunHooks('OasisSkinAssetGroups', array(&$assetGroups));
 
 		// add groups queued via OasisController::addSkinAssetGroup
 		$assetGroups = array_merge($assetGroups, self::$skinAssetGroups);
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Use "static" keyword instead of "self". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS2037)